packages/pxweb2-ui/tsconfig.lib.json

```diff
@@ -4,8 +4,6 @@
     "outDir": "./dist/out-tsc",
     "types": [
       "node",
-      "@nx/react/typings/cssmodule.d.ts",
```

Comment:
Do we not get errors when this is removed? If that is the case. Great! Remove it 👍

Reply:
Yea, I didn't see any errors. And i could build, test and run it fine so :)